.secrets.baseline

```diff
@@ -57,7 +57,154 @@
       "name": "TwilioKeyDetector"
     }
   ],
-  "results": {},
+  "results": {
+    "go.sum": [
```

Comment:
we should exclude `go.sum` in `.pre-commit-config.yaml` and re-run `detect-secrets`